build.sbt

```diff
@@ -93,7 +93,7 @@ dockerCommands := dockerCommands.value.flatMap {
       Cmd("ENV", s"STYLELINT_CONFIG_BASEDIR $defaultDockerInstallationPath/node_modules"),
       cmd,
       Cmd("RUN", s"mv $defaultDockerInstallationPath/docs /docs"),
-      ExecCmd("RUN", Seq("chown", "-R", s"docker:docker", "/docs"): _*),
+      Cmd("RUN", "chown", "-R", "docker:docker", "/docs"),
```

Comment:
You need to use a single string with `Cmd`. You need to remove the `ExecCmd` import too

Reply:
According to the methods documentation, ExecCmd is for executable commands, such as RUN, while Cmd is for arbitrary commands, such as ENV.
Replaced Cmd with ExecCmd for RUN commands.